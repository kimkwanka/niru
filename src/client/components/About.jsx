import React from 'react';
import Helmet from 'react-helmet';

const About = () => (
  <div className="container margin-top-small">
    <Helmet title="About" />
    <h1>About</h1>
    <p>Imagine there was a fancy About page here...</p>
  </div>
);

export default About;
