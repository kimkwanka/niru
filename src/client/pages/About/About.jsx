import React from 'react';
import Helmet from 'react-helmet';

const About = () => (
  <div>
    <Helmet title="About" />
    <h1>About</h1>
  </div>
);

export default About;
