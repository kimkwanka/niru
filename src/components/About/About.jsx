import React from 'react';
import Helmet from 'react-helmet';

const About = () => (
  <div className="container margin-top-small">
    <Helmet title="About" />
    <h1>About</h1>
    <img className="width-100" src="/fistbump.jpg" alt="fistbump" />
    <p>Imagine there was a fancy About page here...</p>
  </div>
);

export default About;
