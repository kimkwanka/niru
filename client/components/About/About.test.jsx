import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import About from './About';

describe('<About /> - Snapshot', () => {
  it('matches its snapshot', () => {
    const wrapper = shallow(<About />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
