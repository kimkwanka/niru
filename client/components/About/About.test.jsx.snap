// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> - Snapshot matches its snapshot 1`] = `
<div
  className="container margin-top-small"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
    title="About"
  />
  <h1>
    About
  </h1>
  <img
    alt="fistbump"
    className="width-100"
    src="fistbump.jpg"
  />
  <p>
    Imagine there was a fancy About page here...
  </p>
</div>
`;
